recipe/meta.yaml

```diff
@@ -41,16 +41,16 @@ test:
 outputs:
   - name: cuda-cccl
     requirements:
-      build:
-        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+      # build:
+      #   - arm-variant * {{ arm_variant_type }}  # [aarch64]
       host:
         - cuda-version {{ cuda_version }}
       run:
         - {{ pin_compatible("cuda-version", max_pin="x.x") }}
         - cuda-cccl_{{ target_platform }} =={{ version }}
         - cuda-cccl-impl =={{ impl_version }}
-      run_constrained:
-        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+      # run_constrained:
```

Comment:
I believe we could keep the run_constrained. Even though we do not support tegra, and do not provide arm-variant, keeping it would help in cases a user mix and match with conda-forge.

Reply:
agreed, included